src/libraries/Common/src/Interop/BSD/System.Native/Interop.Sysctl.cs

```diff
@@ -75,10 +73,24 @@ private static unsafe void Sysctl(int* name, int name_len, ref byte* value, ref
                 {
                     Marshal.FreeHGlobal((IntPtr)value);
                 }
-                throw new InvalidOperationException(SR.Format(SR.InvalidSysctl, *name, Marshal.GetLastPInvokeError()));
+                throw MkException(name, name_len);
             }
 
-            len = (int)bytesLength;
+            len = (uint)bytesLength;
+        }
+
+        private static unsafe InvalidOperationException MkException(int* name, uint name_len)
```

Comment:
We don't use such abbreviation in method names.

Also, to improve performance, the throw expression should be in the helper method if used.